d2/src/main/java/com/linkedin/d2/balancer/servers/ZooKeeperConnectionManager.java

```diff
@@ -184,6 +184,75 @@ protected None convertResponse(None none) throws Exception
     }
   }
 
+  /**
+   * deprecated, use {@link  ConnectionManager#markDownAllServers(Callback)} instead.
+   */
+  @Deprecated
+  public void markDownAllServers(final Callback<None> callback)
```

Comment:
also needs "@Override", right? same for markUpAllServers

Reply:
Actually. It no needs to add override, the java will get this automatically. But it's good to point it out

Reply:
Yeah, just better for code readers.